metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -72,10 +72,6 @@ def __init__(
         extra_headers: Optional[Dict[str, str]] = None,
         ca_certificate_path: Optional[str] = None,
     ):
-        if ":9002" in gms_server:
-            logger.warning(
-                "the rest emitter should connect to GMS (usually port 8080) instead of frontend"
```

Comment:
Not necessarily - going forward, datahub-frontend can serve as a proxy to GMS. People can point to datahub-frontend instead to minimize the access points they need to maintain.